utils/syscall.c

```diff
@@ -382,9 +382,342 @@ static const myst_syscall_pair_t _pairs[] = {
+      SYS_timerfd_settime,
+      SYS_timerfd_gettime,
+      SYS_signalfd4,
+      SYS_eventfd2,
+      SYS_epoll_create1,
+      SYS_dup3,
+      SYS_pipe2,
+      SYS_inotify_init1,
+      SYS_preadv,
+      SYS_pwritev,
+      SYS_perf_event_open,
+      SYS_fanotify_init,
+      SYS_fanotify_mark,
+      SYS_name_to_handle_at,
+      SYS_open_by_handle_at,
+      SYS_syncfs,
+      SYS_setns,
+      SYS_finit_module,
+      SYS_renameat2,
+      SYS_memfd_create,
+      SYS_kexec_file_load,
+      SYS_bpf,
+      SYS_execveat,
+      SYS_userfaultfd,
+      SYS_copy_file_range,
+      SYS_preadv2,
+      SYS_pwritev2,
+      SYS_statx}},
+    {"file",
+     60,
+     {SYS_stat,
+      SYS_open,
+      SYS_lstat,
+      SYS_access,
+      SYS_execve,
+      SYS_truncate,
+      SYS_getcwd,
+      SYS_chdir,
+      SYS_rename,
+      SYS_mkdir,
+      SYS_rmdir,
+      SYS_creat,
+      SYS_link,
+      SYS_unlink,
+      SYS_symlink,
+      SYS_readlink,
+      SYS_chmod,
+      SYS_chown,
+      SYS_lchown,
+      SYS_utime,
+      SYS_mknod,
+      SYS_uselib,
+      SYS_statfs,
+      SYS_pivot_root,
+      SYS_chroot,
+      SYS_acct,
+      SYS_mount,
+      SYS_umount2,
+      SYS_swapon,
+      SYS_swapoff,
+      SYS_quotactl,
+      SYS_setxattr,
+      SYS_lsetxattr,
+      SYS_getxattr,
+      SYS_lgetxattr,
+      SYS_listxattr,
+      SYS_llistxattr,
+      SYS_removexattr,
+      SYS_lremovexattr,
+      SYS_utimes,
+      SYS_inotify_add_watch,
+      SYS_openat,
+      SYS_mkdirat,
+      SYS_mknodat,
+      SYS_fchownat,
+      SYS_futimesat,
+      SYS_newfstatat,
+      SYS_unlinkat,
+      SYS_renameat,
+      SYS_linkat,
+      SYS_symlinkat,
+      SYS_readlinkat,
+      SYS_fchmodat,
+      SYS_faccessat,
+      SYS_utimensat,
+      SYS_fanotify_mark,
+      SYS_name_to_handle_at,
+      SYS_renameat2,
+      SYS_execveat,
+      SYS_statx}},
+    {"ipc",
+     12,
+     {SYS_shmget,
+      SYS_shmat,
+      SYS_shmctl,
+      SYS_semget,
+      SYS_semop,
+      SYS_semctl,
+      SYS_shmdt,
+      SYS_msgget,
+      SYS_msgsnd,
+      SYS_msgrcv,
+      SYS_msgctl,
+      SYS_semtimedop}},
+    {"network", 21, {SYS_sendfile,    SYS_socket,     SYS_connect,
+                     SYS_accept,      SYS_sendto,     SYS_recvfrom,
+                     SYS_sendmsg,     SYS_recvmsg,    SYS_shutdown,
+                     SYS_bind,        SYS_listen,     SYS_getsockname,
+                     SYS_getpeername, SYS_socketpair, SYS_setsockopt,
+                     SYS_getsockopt,  SYS_getpmsg,    SYS_putpmsg,
+                     SYS_accept4,     SYS_recvmmsg,   SYS_sendmmsg}},
+    {"process",
+     14,
+     {SYS_clone,
+      SYS_fork,
+      SYS_vfork,
+      SYS_execve,
+      SYS_exit,
+      SYS_wait4,
+      SYS_kill,
+      SYS_rt_sigqueueinfo,
+      SYS_tkill,
+      SYS_exit_group,
+      SYS_tgkill,
+      SYS_waitid,
+      SYS_rt_tgsigqueueinfo,
+      SYS_execveat}},
+    {"signal",
+     15,
```

Comment:
Should be 14, excluding the commented out line

Reply:
Changed this, thanks